server/leafnode.go

```diff
@@ -125,6 +125,55 @@ func validateLeafNode(o *Options) error {
 	if err := validateLeafNodeAuthOptions(o); err != nil {
 		return err
 	}
+	// In local config mode, check that leafnode configuration refers to accounts that exist.
+	if len(o.TrustedOperators) == 0 {
+		accNames := map[string]struct{}{}
+		for _, a := range o.Accounts {
+			accNames[a.Name] = struct{}{}
+		}
+		// global account is always created
+		accNames[DEFAULT_GLOBAL_ACCOUNT] = struct{}{}
+		accNames[_EMPTY_] = struct{}{} // in the context of leaf nodes, empty account means global account
```

Comment:
Please move the comment before the `accNames[_EMPTY_] = ...` line.

Reply:
fixed